Error: Cannot find module './configuration'

Hi,

I'm trying to use this package to create an incident in Zenduty and running into the following error:

```
Error: Cannot find module './configuration'
Require stack:
- /home/runner/work/action-cve/action-cve/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at new require (internal/modules/cjs/helpers.js:74:18)
    at Object.976 (/home/runner/work/action-cve/action-cve/webpack:/@kunalnagarco/action-cve/node_modules/zenduty-sdk/zenduty/request.js:3:1)
    at __nccwpck_require__ (/home/runner/work/action-cve/action-cve/webpack:/@kunalnagarco/action-cve/webpack/bootstrap:21:1)
    at Object.8630 (/home/runner/work/action-cve/action-cve/webpack:/@kunalnagarco/action-cve/node_modules/zenduty-sdk/zenduty/api_client.js:2:1)
    at __nccwpck_require__ (/home/runner/work/action-cve/action-cve/webpack:/@kunalnagarco/action-cve/webpack/bootstrap:21:1)
    at Object.3261 (/home/runner/work/action-cve/action-cve/webpack:/@kunalnagarco/action-cve/node_modules/zenduty-sdk/zenduty/api/teams_api.js:2:1)
    at __nccwpck_require__ (/home/runner/work/action-cve/action-cve/webpack:/@kunalnagarco/action-cve/webpack/bootstrap:21:1)
```

You can see the error [here](https://github.com/kunalnagarco/action-cve/runs/4584001225?check_suite_focus=true)

Here is the [API call](https://github.com/kunalnagarco/action-cve/pull/45/commits/414f50c62b7480ed4fc98cb7fb71554509a0c9a6#diff-051f8c87d9e802b10401c033d8ab56d5ebd87c9c5e76f191caacb71040eee8cf) that I am making. Kindly advise, thanks!

--- a/zenduty/request.js
+++ b/zenduty/request.js
@@ -1,6 +1,6 @@
 //import fetch for performing various requests to api
 const {fetch}=require("fetch-ponyfill")();
-const conf=new require("./configuration")
+
 class FetchClientObject {
 
      constructor () {

